config/credentials.yml.enc

```diff
@@ -1 +1 @@
-vupay2gL0aBP9GH13Z+rW9ZFm3eS6Bw9CjRHSYb7VWd/i8iOEyKzNhZCBvQuYB4apx0iYtoXdRNnMNTwwAOtMxa29mECF+D9KQps7agUm8Z1LhqKD/WuBKyPpqz6JtLKUxwBoO2oHbN/XgpiRn6rIBqLzWKq7cb1mySSarYzdcPQ49lEIVj+0F2v37XlCu10X5UI8D/2gEd8a8fdpindsUPVJsLvld/aAyhPWNSFZMYSGf7zuXuV8gHjoo0xvQMYmaese6LBA9nTxLbfIXsOdXGdctuNSAp6nbMgvvDIuzqELVe6IMTlhVNI8inE2EoPPqKwQYhDx62enhgAivosy/Qx0f2d23oaY0872iZioarOsk5wt0xZsT261rXs+cPLUlkdU6tEB5SPsJB20IzmsPf6s0TW7itiB7ww--jCSGIJK5396C0dki--+xKE3Cjd48u68nRFeo4/fg==
\ No newline at end of file
+XJ8lk1X9JAOeX8mTmIVHs0FHg6UFx6Y0FQAvK9o+u2Yrhy9tZ13bHYPy7uj51SWZ2abnos1cee+FAPoAqN8ZQxna+n0CLbqgLPkotWfCeujwlcW4eqNqDq0RxOoZ5yvvZgexDzgacr34C8CHiGF3NPgiptmAehpkgwSxPkVTCTksI2IcerprKiv1zFks/jcEno3knj8uNj38qsASzu6yQ5LlbRzZqna5gitAm4W43tHsbEpEh+Ii7Jgdb0MPQ0KgMzA0Pf4NBVlegF5gylCjgWvNXNHMfOaElH7OPeMbbpNWnDxkHVHfNiO3QB2TNbIL37QbacfN7FAzF7KqyHbwTCDXIn8NbrZXKhaw9bvga9VIYHYg45JpjlhSr93nQoJMKsehsRWsC+uGDDRIIr2DiBMTSJKwGkgB2W6QYtEovb7O87uOy/BtKMmZEqYwOv+NQXGdAKEflD+mgmd3obOBGut1/c1cOCcWAJOfbgfL2MfEmR6SIc2dGU+ZPTGup7O0nTV3HjKbc81Urli3pRsHT0A2ywq46Jg4onEqiOrWBzN/8hjlbiriz6/e9SxpgHvulHdXkg4bZMQjRJOXoaed7q7rM3whxZdv/Cs7s5UDvmOXJ0hr0CTN270T0F2drDGmz9+eK2LbnztRN3Qap2Y=--zAN+1m00iGiTKdSX--d5ZA2Oc/cOkKcTMCOPXqiw==
```

Comment:
What did we have to add to the credentials?

Reply:
Also, do you have the `config/master.key`? Do we need to share this across the team somehow?